csharp/src/Ice/Communicator.cs

```diff
@@ -1011,6 +1038,14 @@ async Task PerformDisposeAsync()
         public IEndpointFactory? IceFindEndpointFactory(Transport transport) =>
             _transportToEndpointFactory.TryGetValue(transport, out IEndpointFactory? factory) ? factory : null;
 
+        // Add one or more dispatch interceptors, only to use by PluginInitializationContext
+        internal void AddDispatchInterceptor(params DispatchInterceptor[] interceptors) =>
```

Comment:
I am pretty sure we don't need params for these methods since they are not called directly.

Reply:
Fixed